Makefile

```diff
@@ -9,5 +9,8 @@ build: Dockerfile
 test:
 	docker run -i -t --rm $(IMAGE) klist 2>&1 | grep "klist: No credentials"
 
+ci-test:
```

Comment:
Do we actually need an extra target? Perhaps just remove `-t` from the regular `make test`...